pkg/agent/cniserver/interface_configuration_linux.go

```diff
@@ -126,6 +128,26 @@ func moveVFtoContainerNS(vfNetDevice string, containerID string, containerNetNS
 			return fmt.Errorf("failed to configure IP address for container %s: %v", containerID, err)
 		}
 		klog.V(2).Infof("ipam.ConfigureIface result: %+v, err: %v", result, err)
+		// For VF interface in pod namespace (without VF Representator), configure policy rules to isolate the route lookup specific to the secondary network.
```

Comment:
Yes. But we need to isolate secondary network traffic from primary network, to avoid secondary network traffic to use default route from primary. 
with the configured secondary network, we will have
1) create IP rule to divert all the secondary network specific route lookup to custom table (200).
2) At custom route table, add default route to push the traffic towards eth2 (secondary network).

Reply:
But you do not configure a gateway for secondary network, and so I think even without the ip rule and route, traffic to the secondary network subnet will still go from the secondary interface anyway, no?